apps/admin-gui/src/app/shared/components/dialogs/applications-list-columns-change-dialog/applications-list-columns-change-dialog.component.ts

```diff
@@ -74,32 +101,51 @@ export class ApplicationsListColumnsChangeDialogComponent implements OnInit {
               this.attribute = attribute;
               const configuredColumns: string[] = attribute.value as string[];
               if (configuredColumns !== null && configuredColumns.length > 0) {
-                this.selectedColumns.setValue(configuredColumns);
+                this.selectedColumns = configuredColumns;
               } else {
-                this.selectedColumns.setValue(this.simpleColumns);
+                this.selectedColumns = this.simpleColumns;
               }
             });
+          this.loading = false;
         });
     }
   }
 
   confirm(): void {
+    if (
+      this.selectedColumns === null ||
+      this.selectedColumns.length === 0 ||
+      containsExactlyInAnyOrder(this.selectedColumns, this.simpleColumns)
+    ) {
+      this.attribute.value = [];
+    }
     if (this.data.groupId) {
       this.changeGroupAttribute();
     } else {
       this.changeVoAttribute();
     }
   }
+
   cancel(): void {
     this.dialogRef.close(false);
   }
   default(): void {
```

Comment:
Give this method a little bit of space.
```suggestion
  cancel(): void {
    this.dialogRef.close(false);
  }
  
  default(): void {
```